packages/editor-ui/src/components/Error/NodeErrorView.vue

```diff
@@ -428,7 +433,7 @@ async function onAskAssistantClick() {
 				v-n8n-html="getErrorDescription()"
 			></div>
 			<div
-				v-if="isAskAssistantAvailable"
+				v-if="isAskAssistantAvailable && !nodeIsHidden()"
```

Comment:
I think this should all be part of `isAskAssistantAvailable` so we don't add extra logic to Vue templates.

Reply:
Good idea 🙏